testData/solutions/win32.json5

```diff
@@ -1,4 +1,6243 @@
+                                "VocalizerExpressive-French",
+                                "VocalizerExpressive-Dutch",
+                                "VocalizerExpressive-Arabic",
+                                "VocalizerExpressive-Portuguese",
+                                "VocalizerExpressive-Czech",
+                                "VocalizerExpressive-Danish",
+                                "VocalizerExpressive-Hebrew",
+                                "VocalizerExpressive-Hungarian",
+                                "VocalizerExpressive-Italian",
+                                "VocalizerExpressive-Korean",
+                                "VocalizerExpressive-Norwegian",
+                                "VocalizerExpressive-Polish",
+                                "VocalizerExpressive-Romanian",
+                                "VocalizerExpressive-Slovak"
+                            ],
+                            "default": "VocalizerExpressive-English"
+                        }
+                    },
+                    "ZtSpeech.Language": {
+                        "schema": {
+                            "title": "Language id",
+                            "description": "Language code identifier.",
+                            "enum": [
+                                9,
+                                7,
+                                10,
+                                12,
+                                1033,
+                                19,
+                                1,
+                                22,
+                                5,
+                                6,
+                                11,
+                                13,
+                                14,
+                                16,
+                                18,
+                                20,
+                                24,
+                                27
+                            ],
+                            "enumLabels": [
+                                "msmobile-English",
+                                "German",
+                                "Spanish",
+                                "French",
+                                "English",
+                                "Dutch",
+                                "Arabic",
+                                "Portuguese",
+                                "Czech",
+                                "Danish",
+                                "Finnish",
+                                "Hebrew",
+                                "Hungarian",
+                                "Italian",
+                                "Korean",
+                                "Norwegian",
+                                "Romanian",
+                                "Slovak"
+                            ],
+                            "default": 9
+                        }
+                    },
+                    // VOICES SECTION
```

Comment:
I really think we should get rid of the voice-specific tweaks, and only keep things that can be configured generally for all voices.

Reply:
it probably makes sense, but for the moment I'm leaving them there similarly to how we did for the JAWS pull from last week.

Reply:
It's fine, I have thoughts about using the power of IoC to represent these more simply, like reuse the common settings for each language.  I can revisit in the LSR work.